DOCS-308, 338, 345, 350, 354, 404, 405

Lmariomini opened the pull request:
Updated to include: Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost; changes in delete-ability of request and notice policies. Modified 'Fixed due date schedule (due date limit) language.

Lmariomini committed:
DOCS-338, DOCS-344 and misc. changes. Not done DOCS-350 or 355 yet.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354

Updated with bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees ; Limiting of request pickup SPs; Notify user when aged-to-lost.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354

Updated with bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of request pickup SPs; Notify user when aged-to-lost. Added also clarifying language that items are not counted against the ‘max number of overdue items’ patron block while they are within the grace period.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354 

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost. Modified 'Fixed due date schedule (due date limit) language.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost. Modified 'Fixed due date schedule (due date limit) language.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost. Modified 'Fixed due date schedule (due date limit) language.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost. Modified 'Fixed due date schedule (due date limit) language.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354 

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost. Modified 'Fixed due date schedule (due date limit) language.

Lmariomini committed:
Merge branch 'folio-org:poppy' into poppy-settings-circ

Lmariomini committed:
DOCS-308, 338, 345, 350, 354, 404, 405

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost; deletion of request and notice policies. Modified 'Fixed due date schedule (due date limit) language.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354, 404, 405

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost; changes to delete-ability of request and notice policies. Modified 'Fixed due date schedule (due date limit) language.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354, 404, 405

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost; changes in delete-ability of Request and Notice policies. Modified 'Fixed due date schedule (due date limit) language.

Lmariomini committed:
Merge branch 'folio-org:poppy' into poppy-settings-circ

julianladisch commented on content/en/docs/Settings/Settings_circulation/settings_circulation.md:
```diff
@@ -6,6 +6,8 @@ weight: 40
 tags: ["subtopic"]   
 ---
 
+<table style="border: 2px solid black"><tr><td><strong> This section of the documentation contains links to external sites. Please be advised that these sites are not maintained by the FOLIO Documentation Group and may not be aligned with the current release of FOLIO.</strong></td></tr></table>
```
`<table>` for non-tabluar data should be avoided if possible.

* For tabular data use the Markdown table syntax: https://github.github.com/gfm/#tables-extension-
* For non-tabular data use regular paragraphs.

Drawbacks of `<table>`:
* The inner text doesn't support Markdown.
* Other editors of the documentation may not understand HTML syntax.
* Requires effort to keep it accessible:
  * https://www.w3.org/WAI/WCAG22/Techniques/failures/F46.html
  * https://www.w3.org/WAI/WCAG22/Techniques/failures/F49.html
  * https://www.w3.org/WAI/WCAG22/Techniques/failures/F91.html

In this case a simple paragraph put into `**` is sufficient.

Lmariomini replied:
I'm sure you're correct, but I was copy-and-pasting the suggestion made by a fellow Docs WG member in our meeting on Nov. 8. I suggest you post your alternative recommendation in the Docs Slack documentation-wg.

julianladisch commented on content/en/docs/Settings/Settings_circulation/settings_circulation.md:
```diff
@@ -6,6 +6,8 @@ weight: 40
 tags: ["subtopic"]   
 ---
 
+<table style="border: 2px solid black"><tr><td><strong> This section of the documentation contains links to external sites. Please be advised that these sites are not maintained by the FOLIO Documentation Group and may not be aligned with the current release of FOLIO.</strong></td></tr></table>
```
A reader may not consider `wiki.folio.org` an external site.

A reader may miss this warning at the beginning.

The easiest solution for the reader is putting this warning next to the affected link: "That site may be aligned with a different FOLIO release because it is not maintained by the FOLIO Documentation Group."

Lmariomini replied:
Putting a warning at the top of the sections with external links was the agreed-upon way in the Docs WG meeting of November 8.

julianladisch commented on content/en/docs/Settings/Settings_circulation/settings_circulation.md:
```diff
@@ -6,6 +6,8 @@ weight: 40
 tags: ["subtopic"]   
 ---
 
+<table style="border: 2px solid black"><tr><td><strong> This section of the documentation contains links to external sites. Please be advised that these sites are not maintained by the FOLIO Documentation Group and may not be aligned with the current release of FOLIO.</strong></td></tr></table>
```
"current" should be avoided because it is a moving target. When Quesnelia gets released it becomes the current version. What about "may be aligned with a different FOLIO release"?

Lmariomini replied:
The wording used was what was agreed upon in the meeting of Nov. 8 (see https://wiki.folio.org/pages/viewpage.action?pageId=139102866) . I can change the wording, but you might want to post your suggestion in the documentation-wg slack channel.

Lmariomini committed:
DOCS-308, 338, 345, 350, 354, 404, 405 (Settings>Circ)

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost; Changes in delete-ability of request and patron notice policies. Modified 'Fixed due date schedule (due date limit) language. Added agreed-upon external links warning, with Julian's suggested changes.

Lmariomini committed:
Merge branch 'folio-org:poppy' into poppy-settings-circ

Lmariomini committed:
DOCS-308, 338, 345, 350, 354, 404, 405 (settings>circ)

Updated to include Bundling of overnight aged-to-lost; Circ rules followed by TLR holds; Reminder fees; Limiting of Request pickup SPs; Notify user when aged-to-lost; changes to delete-ability of request and otice policies. Modified 'Fixed due date schedule (due date limit) language. Added warning notice for external links using Julian's suggested changes.
